code/api/src/modules/user/mutations.js

```diff
@@ -27,6 +27,22 @@ export const userSignup = {
   resolve: create
 }
 
+// User update
+export const userUpdate = {
```

Comment:
The userUpdate method takes two arguments, a user ID and a user Stye (string `'casual, classy, casual...'`)